browser/ipfs/ipfs_tab_helper.cc

```diff
@@ -39,6 +65,21 @@ void SetupIPFSProtocolHandler(const std::string& protocol) {
   base::MakeRefCounted<shell_integration::DefaultProtocolClientWorker>(protocol)
       ->StartCheckIsDefault(base::BindOnce(isDefaultCallback, protocol));
 }
+
+// Expects dns TXT record in format: name=value
+std::string GetDNSRecordValue(const std::vector<std::string>& text_results,
```

Comment:
is there no existing code to parse TXT records in chromium?

Reply:
no, it's just a list of strings, each can be in any format

Reply:
ah, right. I forgot it has no specified format